src/main/java/net/minestom/server/world/biomes/BiomeManager.java

```diff
@@ -14,13 +16,20 @@
 /**
  * Allows servers to register custom dimensions. Also used during player joining to send the list of all existing dimensions.
  * <p>
- * Contains {@link Biome#PLAINS} by default but can be removed.
  */
 public final class BiomeManager {
     private final Map<Integer, Biome> biomes = new ConcurrentHashMap<>();
```

Comment:
id field should be removed from Biome/BiomeImpl, and id should only be known by BiomeManager.